examples/axisartist/simple_axisartist1.py

```diff
@@ -1,26 +1,51 @@
 """
-==================
-Simple Axisartist1
-==================
+=============================
+Custom spines with axisartist
+=============================
 
+This example showcases the use of :mod:`.axisartist` to draw spines at custom
+positions (here, at ``y = 0``).
+
+Note, however, that it is simpler to achieve this effect
+using standard `.Spine` methods, as demonstrated in
+:doc:`/gallery/ticks_and_spines/centered_spines_with_arrows`.
+
+.. redirect-from:: /gallery/axisartist/simple_axisline2
 """
+
 import matplotlib.pyplot as plt
 from mpl_toolkits import axisartist
+import numpy as np
+
+
+fig = plt.figure(figsize=(6, 3), constrained_layout=True)
+gs = fig.add_gridspec(1, 2)
```

Comment:
Do you have to use gridspec for this?

Reply:
The other option is to use `add_subplot(211, axes_class=...)` or `add_subplot(2, 1, 1, axes_class=...)` but I'm not sure we really like that...

Reply:
Ok, so maybe a note about why you can't use the axes factory methods (subplots/subplots mosaic) because this is a different axes type? I'm not sure how to phrase this, but it's a thing that always throws me for a loop.

Reply:
sure

Reply:
Alternative would be to make two separate plots so that you can `add_subplot(axes_class=...)`.

Reply:
I tend to think that showcasing everything on a single figure makes things easier to compare, but I don't feel strongly either way.

Reply:
Let's not overthink this and merge as an improvement. We can always polish more later.